### DISCUSSION TITLE
Blazor Demo

### DISCUSSION THREAD

**Discussion by fundez** [MarimerLLC/csla#1833] [Questions] [answered]
I noticed the blazor demo was removed from the project tracker and that the stand-alone blazor demo is setup to run in client mode only and no longer has a project named UI. Is this the new recommended CSLA blazor design?

Also, the blazor demo project in 5.31 still uses the old style vm.RefreshAsync. Are there plans to update it. If not I should be able to update it myself.

Thanks,

Mike

**Comment by rockfordlhotka** [comment]
I'm confused, as I see the 3 Blazor projects in the ProjectTracker solution in v5.3.1

![image](https://user-images.githubusercontent.com/2333134/95510582-2d94ed80-0984-11eb-83fe-dfd2bce37b30.png)

**Reply by rockfordlhotka** [reply]
Also, fwiw, .NET 5 will have some new project templates, and I think they are including one that is a dual-deploy solution (wasm and server-side).

**Comment by fundez** [comment, marked as answer]
Sorry for the confusion, Somehow I managed to download 5.1 last week. I must of done something stupid.

Thanks for the clarification.

Mike
